pallets/identity/src/lib.rs

```diff
@@ -666,21 +663,6 @@ decl_error! {
 }
 
 impl<T: Config> Module<T> {
-    /// Only used by `create_asset` since `AssetDidRegistered` is defined here instead of there.
-    pub fn commit_token_did(did: IdentityId, ticker: Ticker) {
-        DidRecords::<T>::insert(did, DidRecord::default());
-        Self::deposit_event(RawEvent::AssetDidRegistered(did, ticker));
-    }
```

Comment:
If we are dropping support for this, we need to create a ticket to remove the old DidRecords entries for assets.